hack/update-codegen.sh

```diff
@@ -29,14 +30,9 @@ echo "GOPATH=$GOPATH"
 
 group "Kubernetes Codegen"
 
-# generate the code with:
-# --output-base    because this script should also be able to run inside the vendor dir of
-#                  k8s.io/kubernetes. The output-base is needed for the generators to output into the vendor dir
-#                  instead of the $GOPATH directly. For normal projects this can be dropped.
-${CODEGEN_PKG}/generate-groups.sh "deepcopy,client,informer,lister" \
-  "knative.dev/eventing-ceph/pkg/client" "knative.dev/eventing-ceph/pkg/apis" \
-  "sources:v1alpha1" \
-  --go-header-file ${REPO_ROOT_DIR}/hack/boilerplate.go.txt
+kube::codegen::gen_helpers \
```

Comment:
You need to create the clients too (via `kube::codegen::gen_client`). Check for example on https://github.com/knative/eventing/blob/07d379056bd898134b8b5021e66590cea0f16397/hack/update-codegen.sh#L40-L45